build.test-scenes.json

```
@@ -20,13 +20,19 @@
   {
     "name": "Compile test scenes (gamekit, part B)",
     "kind": "Webpack",
-    "file": "./public/test-parcels/!(1)*/game.ts",
+    "file": "./public/test-parcels/!(1|-100)*/game.ts",
```

```suggestion
    "file": "./public/test-parcels/!(1|-100|6)*/game.ts",
```
Avoid `6*` (hellmap)